src/Static.ts

```diff
@@ -8,18 +8,37 @@ export class Static {
     }
 
     public static buildSectionBlock(mrkdwnValue: string): SectionBlock {
-        return { type: "section", text: { type: "mrkdwn", text: mrkdwnValue } };
+        return {type: "section", text: {type: "mrkdwn", text: mrkdwnValue}};
     }
 
     public static buildContextBlock(mrkdwnValue: string): ContextBlock {
-        return { type: "context", elements: [ { type: "mrkdwn", text: mrkdwnValue } ] };
+        return {type: "context", elements: [{type: "mrkdwn", text: mrkdwnValue}]};
```

Comment:
Why did all this spacing change?

Reply:
I believe this was due to running ESLint. I did not touch these functions.